go.mod

```diff
@@ -134,6 +132,19 @@ require (
 	sigs.k8s.io/yaml v1.4.0
 )
 
+require (
+	github.com/bytedance/sonic v1.11.1 // indirect
+	github.com/chenzhuoyu/base64x v0.0.0-20230717121745-296ad89f973d // indirect
+	github.com/cloudwego/configmanager v0.2.0 // indirect
+	github.com/cloudwego/dynamicgo v0.2.0 // indirect
+	github.com/cloudwego/localsession v0.0.2 // indirect
+	github.com/fatih/structtag v1.2.0 // indirect
+	github.com/iancoleman/strcase v0.2.0 // indirect
+	github.com/klauspost/cpuid/v2 v2.2.4 // indirect
+	github.com/modern-go/gls v0.0.0-20220109145502-612d0167dce5 // indirect
+	github.com/twitchyliquid64/golang-asm v0.15.1 // indirect
+)
```

Comment:
Nit: please make sure there's a single block for all indirect deps, to keep them sorted